### PR TITLE
feat(protocol): Add scraping_attempts field

### PR DESCRIPTION
**Pull request by jjbayer**
https://github.com/getsentry/sentry/pull/57461 introduced a new top-level field on the event that is only written by sentry. To prevent errors when the event passes through normalization after writing `scraping_attempts`, make sure that the new field is part of the schema known by (lib)relay.

### CODE REVIEW

**Review by olksdr — APPROVED** [review]

File: relay-event-schema/src/protocol/event.rs [hunk 1]
```diff
@@ -480,6 +480,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub breakdowns: Annotated<Breakdowns>,
 
+    /// Information about attempts to scrape a JS source or sourcemap file from the web.
+    /// This field is populated by sentry.
+    #[metastructure(omit_from_schema)] // not part of external schema
+    pub scraping_attempts: Annotated<Object<Value>>,
```
Comment (loewenheim) [inline relay-event-schema/src/protocol/event.rs]: Does `Object` here mean something like a JSON object? The field is actually an array.

Comment (jjbayer) [reply relay-event-schema/src/protocol/event.rs]: Good catch, it's an alias for a `BTreeMap`. I will change it and add a test case.

**Review by loewenheim — APPROVED** [review]

**Review by iker-barriocanal — APPROVED** [review]

File: relay-event-schema/src/protocol/event.rs [hunk 2]
```diff
@@ -480,6 +480,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub breakdowns: Annotated<Breakdowns>,
 
+    /// Information about attempts to scrape a JS source or sourcemap file from the web.
+    /// This field is populated by sentry.
+    #[metastructure(omit_from_schema)] // not part of external schema
+    pub scraping_attempts: Annotated<Array<Value>>,
```
Comment (iker-barriocanal) [inline relay-event-schema/src/protocol/event.rs]: Should we make this a `Value` instead of `Array` instead?

Comment (jjbayer) [reply relay-event-schema/src/protocol/event.rs]: I think this is a good call. Then Relay dies not even have to be updated if the sentry-side schema ever changed to an object to include more metadata, for example. Updated now.

**Review by iker-barriocanal — APPROVED** [review]

**Pull request merged**